### PR TITLE
Update Backup pending screen design and copy

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR updates the Backup pending screen to prevent users from contacting support while the site first backup is being made.

Fixes 1197351014149633-as-1197360555544595

### Testing instructions

- Download the PR
- Run Calypso with `yarn start`
- Run Jetpack cloud concurrently with `yarn start-jetpack-cloud-p`
- Create a new self-hosted Jetpack site, and purchase Backup or a plan that includes Backup
- Visit `/backup/:site`
- Check that you see the new screen in both environments
- Check that the copy matches the captures below (watch out the difference between cloud and Calypso), and that the links redirect to the proper places

#### Jetpack cloud
_Desktop_
Screen Shot 2020-10-06 at 1 14 41 PM
_Mobile_
Screen Shot 2020-10-06 at 1 15 00 PM

#### Calypso
_Desktop_
Screen Shot 2020-10-06 at 1 14 05 PM

_Mobile_
Screen Shot 2020-10-06 at 1 14 15 PM

### CODE REVIEW

File: client/components/jetpack/daily-backup-status/status-card/no-backups-yet.jsx [hunk 1]
```diff
@@ -1,28 +1,59 @@
...
+				<h2 className="status-card__title">{ translate( 'First backup is on the way!' ) }</h2>
+				<div className="status-card__label">
+					{ translate(
+						'And it should become available in the {{strong}}next 24 hours{{/strong}}. We will let you know once the backup has been completed.',
+						{ components: { strong: <strong /> } }
+					) }
+				</div>
+				<ul className="status-card__link-list">
+					<li>
+						<ExternalLink href={ siteUrl }>{ translate( 'Visit your website' ) }</ExternalLink>
```
Comment (a8ci18n) [inline client/components/jetpack/daily-backup-status/status-card/no-backups-yet.jsx:48]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Visit Your Website' )` *ES Score: 8*

File: client/components/jetpack/daily-backup-status/status-card/style.scss [hunk 2]
```diff
@@ -96,20 +106,11 @@
...
-	.activity-card
-	.card
-	.activity-log-item__actor
-	.gravatar,
-.status-card__realtime-details-card
-	.activity-card
-	.card
-	.activity-log-item__actor
-	svg {
+.status-card__realtime-details-card .activity-card .card .activity-log-item__actor .gravatar,
```
Comment (monsieur-z) [inline client/components/jetpack/daily-backup-status/status-card/style.scss:109]: Just some automatic formatting.

**Review by elliottprogrammer — APPROVED** [review]
Looks and works great.
LGTM! 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/4839133

Thank you @monsieur-z for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/4839133).

**Pull request merged**